srcalc/src/main/java/gov/va/med/srcalc/domain/model/ValueMatcher.java

```diff
@@ -96,9 +102,63 @@ public Expression getParsedExpression()
         return fBooleanExpression;
     }
     
+    /**
+     * Returns true if this matcher should be evaluated and false if it should not
+     * be evaluated.
+     */
+    @Basic
+    public boolean isExpressionEnabled()
+    {
+        return fExpressionEnabled;
+    }
+    
+    public void setExpressionEnabled(final boolean expressionEnabled)
+    {
+        fExpressionEnabled = expressionEnabled;
+    }
+    
+    /**
+     * Evaluate this ValueMatcher's boolean expression based on the given context and value.
+     * If the apply condition flag is false, this method will always return true.
+     * @param context the EvaluationContext used to evaluate this matcher against
+     * @param value the value for this matcher's variable
+     * @return the boolean value to which the expression evaluates
+     */
     public boolean evaluate(final EvaluationContext context, final Value value)
     {
-        return fBooleanExpression.getValue(context, value, Boolean.class);
+        if(fExpressionEnabled && !fBooleanExpression.getExpressionString().isEmpty())
```

Comment:
I think we should drop the special case for the empty expression now. It's just confusing. Let it throw an exception like any other invalid expression.